community/cypher/cypher-logical-plans-3.4/src/main/scala/org/neo4j/cypher/internal/v3_4/logical/plans/LogicalPlan.scala

```diff
@@ -21,15 +21,20 @@ package org.neo4j.cypher.internal.v3_4.logical.plans
 
 import java.lang.reflect.Method
 
+import org.neo4j.cypher.internal.ir.v3_4.{PlannerQuery, Strictness}
 import org.neo4j.cypher.internal.util.v3_4.Foldable._
-import org.neo4j.cypher.internal.util.v3_4.{Foldable, InternalException, Rewritable, Unchangeable}
 import org.neo4j.cypher.internal.util.v3_4.Rewritable._
-import org.neo4j.cypher.internal.ir.v3_4.{PlannerQuery, Strictness}
-import org.neo4j.cypher.internal.util.v3_4.attribution.{IdGen, SameId}
+import org.neo4j.cypher.internal.util.v3_4.attribution.{Id, IdGen, SameId}
+import org.neo4j.cypher.internal.util.v3_4.{Foldable, InternalException, Rewritable, Unchangeable}
 import org.neo4j.cypher.internal.v3_4.expressions.Expression
 
+import scala.util.hashing.MurmurHash3
 import scala.collection.mutable
 
+object LogicalPlan {
+  val LOWEST_TX_LAYER = 0
+}
```

Comment:
I think this was added by mistake. I'm going to merge anyway if this get green, and remove that in my next PR.

Reply:
yes it might have snuck in via a rebase conflict